core/src/main/java/com/whylogs/core/SchemaInformation.java

```diff
@@ -8,7 +8,7 @@
 @UtilityClass
 public class SchemaInformation {
   final int SCHEMA_MAJOR_VERSION = 1;
-  final int SCHEMA_MINOR_VERSION = 2;
+  final int SCHEMA_MINOR_VERSION = 3;
 
```

Comment:
Give a schema bump, its nice to have unit tests merging a newly generated profile against an older one in src/test/resources to make sure there aren't any merge issues against older data

Reply:
I added a test for roundtrip parsing + merging - the parsing itself turns an old profile into a new profile in memory FYI